### PR TITLE
Use https for link to contributor covenant

### PR DESCRIPTION
**Pull request by npu21**
https://github.com/kubernetes/website/issues/25835

### CODE REVIEW

File: content/id/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ Kode etik ini berlaku baik di dalam ruang proyek maupun di ruang publik ketika s
 Contoh perilaku kasar, melecehkan, atau tidak dapat diterima di Kubernetes dapat dilaporkan dengan menghubungi [Komite Kode Etik Kubernetes](https://git.k8s.io/community/committee-code-of-conduct) melalui <conduct@kubernetes.io>. Untuk proyek lain, silakan hubungi pengelola proyek CNCF atau mediator kami, Mishi Choudhary <mishi@linux.com>.
 
 Kode Etik ini diadaptasi dari Covenant Contributor
```
Comment (habibrosyad) [inline content/id/community/static/cncf-code-of-conduct.md:25]: Could you update this line as well?

```suggestion
Kode Etik ini diadaptasi dari Contributor Covenant
```

File: content/id/community/static/cncf-code-of-conduct.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ Kode etik ini berlaku baik di dalam ruang proyek maupun di ruang publik ketika s
...
-<http://contributor-covenant.org>, versi 1.2.0, tersedia di
-<http://contributor-covenant.org/version/1/2/0/>
+<https://contributor-covenant.org>, versi 1.2.0, tersedia di
+<https://contributor-covenant.org/version/1/2/0/>
```
Comment (habibrosyad) [inline content/id/community/static/cncf-code-of-conduct.md:27]: Also this, to match the upstream English md

```suggestion
(https://contributor-covenant.org), versi 1.2.0, tersedia di
https://contributor-covenant.org/version/1/2/0/
```

**Review by irvifa — COMMENTED** [review]
Hi @npu21 are you still working on this?
I think after you address @habibrosyad comments this is good to go

**Comment by kbhawkey** [comment]
Hello @npu21 .
This PR looks inactive. Close?

**Pull request closed without merging**